src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -34,7 +34,7 @@ public class BlockStoreQueue : IBlockStoreQueue
     {
         /// <summary>Maximum interval between saving batches.</summary>
         /// <remarks>Interval value is a prime number that wasn't used as an interval in any other component. That prevents having CPU consumption spikes.</remarks>
-        private const int BatchMaxSaveIntervalSeconds = 37;
```

Comment:
Just revert this then we are good to go!

Reply:
Done.